Makefile

```diff
@@ -41,20 +43,25 @@ crystal11_obj := $(crystal_obj:.o=11.o)
 all: crystal
 crystal: pokecrystal.gbc
 crystal11: pokecrystal11.gbc
+crystalvc: pokecrystalvc.patch
```

Comment:
```makefile
crystalvc: pokecrystalvc.gbc pokecrystalvc.patch
```

Reply:
That's implied by `pokecrystalvc.patch: pokecrystalvc.gbc` later in the file, but ok.